src/filters/fs.rs

```diff
@@ -107,7 +107,7 @@ fn path_from_tail(
             future::ready(sanitize_path(base.as_ref(), tail.as_str()))
                 .and_then(|mut buf| async {
                     let clone_buf = buf.clone();
-                    let is_dir = tokio_executor::blocking::run(move || clone_buf.is_dir()).await;
+                    let is_dir = tokio::task::spawn_blocking(move || clone_buf.is_dir()).await.expect("FIXME");
```

Comment:
should this get its own rejection?

Reply:
I think you can [block_in_place](https://docs.rs/tokio/0.2.0/tokio/task/fn.block_in_place.html) as tokio is doing the same within `tokio::fs::` :